R/Visualize_Metric.R

```diff
@@ -133,12 +141,16 @@ Visualize_Metric <- function(
       strType = "Score",
       vThreshold = vThreshold
     )
-
-    lCharts$metricTable <- Report_MetricTable(
-      dfResults = dfResults_latest,
-      dfGroups = dfGroups,
-      strGroupLevel = lMetric$GroupLevel
-    )
+    if(!is.null(dfGroups)) {
+      lCharts$metricTable <- Report_MetricTable(
```

Comment:
@lauramaxwell do you know if `Report_MetricTable()` is meant to be one of the visualizations that can be rendered with just results? If so, I think I need to make some modifications there

Reply:
Ideally, although we should always have participant counts.  Still if it's not too much effort this is a good candidate to optionalize group metadata.